src/Chapter10.Tests/Listing10.13.MakingTypesAvailableOutsideAnAssembly.Tests.cs

```diff
@@ -14,26 +13,15 @@ public class TypeTests
         [TestMethod]
         public void MakingTypesAvailableExternallyPS1_ExitCodeIs0()
         {
-            if (PowerShellTestsUtilities.PowerShellNotInstalled()) Assert.Inconclusive("Powershell not installed");
+            if (PowerShellTestUtilities.PowerShellNotInstalled) Assert.Inconclusive("Powershell not installed");
             //EssentialCSharp\\src\\Chapter10.Tests\\bin\\Debug\\netcoreapp3.0
             string ps1Path = Path.GetFullPath("../../../../Chapter10/Listing10.13.MakingTypesAvailableExternally.ps1", Environment.CurrentDirectory);
```

Comment:
You can use Intellitect.Multitool.RepositoryPaths.GetDefaultRepoRoot() to get the root of the repo